### PR TITLE
Ensure filesystem usage isn't collected for devicemapper storage

### PR DESCRIPTION
**Pull request by jimmidyson**
@vishh Please review when you get a chance.

### CODE REVIEW

**Comment by jimmidyson** [comment]
Fixes #1094

**Comment by zaa** [comment]
Could you please add a comment or note why filesystem usage should not be collected for devicemapper storage?

**Comment by jimmidyson** [comment]
@zaa it should be it's just not implemented yet. Discussion in #959.

File: container/docker/handler.go [hunk 1]
```diff
@@ -137,18 +137,29 @@ func newDockerContainerHandler(
 	// Add the Containers dir where the log files are stored.
 	otherStorageDir := path.Join(path.Dir(storageDriverDir), pathToContainersDir, id)
 
-	rwLayerID, err := getRwLayerID(id, storageDriverDir, dockerVersion)
```
Comment (vishh) [inline container/docker/handler.go]: Is this method failing with devicemapper backend?

Comment (jimmidyson) [inline container/docker/handler.go]: Yes, hence this PR.

**Pull request closed without merging**